Makefile.am

```diff
@@ -3924,10 +3924,10 @@ noinst_LTLIBRARIES += \
 
 src/udev/keyboard-keys-list.txt:
 	$(AM_V_at)$(MKDIR_P) $(dir $@)
-	$(AM_V_GEN)$(CPP) $(CFLAGS) $(AM_CPPFLAGS) $(CPPFLAGS) -dM -include linux/input.h - < /dev/null | $(AWK) '/^#define[ \t]+KEY_[^ ]+[ \t]+[0-9K]/ { if ($$2 != "KEY_MAX") { print $$2 } }' > $@
```

Comment:
this is missing the matching change to src/udev/generate-keyboard-keys-list.sh and the one below to src/udev/generate-keyboard-gperf.py for meson. I'd say the best option here would be to have a separate patch to Makefile.am to use the scripts as well so we only have one place to fix this up.